src/search/components/SearchResult/index.js

```diff
@@ -26,16 +41,35 @@ export default class SearchResult extends React.Component {
   render() {
     const { result } = this.props;
     return (
-      <Link to={`/search/addons/${result.slug}`} className="search-result" ref="container">
-        <div className="search-result--name" ref="name">
-          {result.name}
+      <li className="SearchResult" ref="container">
+        <div>
+          <img
+            className="SearchResult-icon"
+            src={result.icon_url}
+            height="64"
+            width="64"
+            alt="Icon"
+          />
+        </div>
+        <div className="SearchResult-main">
+          <h2 className="SearchResult-heading">
+            <Link to={`/search/addons/${result.slug}`} className="SearchResult-name" ref="name">
+              {result.name}
+            </Link>
+          </h2>
+          <div className="SearchResult-info" ref="guid">{result.guid}</div>
```

Comment:
Should start using non-string refs as soon as possible. We're going to need to replace them all eventually. See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md